extension/src/documents.ts

```diff
@@ -5,9 +5,37 @@ import { AutoLispExt } from './extension';
 import { ProjectTreeProvider  } from "./project/projectTree";
 
 
+enum Origins {
+	OPENED = 'O',
+	WSPACE = 'W',
+	PROJECT = 'P'
+}
+
+
+interface DocumentSources {
+	native: vscode.TextDocument;
```

Comment:
we need to give its more descriptive name, I think `rawDoc` or any others

Reply:
In the context of this not being exported and living inside of `DocumentSources` interface, I actually think native and internal are meaningful. Clearly just an F2 to fix it if your comment isn't assuming use-cases beyond _documents.ts_ though.

Reply:
So, did you come around on this or are we going with roDoc & vsDoc?

Reply:
I am ok with the current name